<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindow</class>
 <widget class="QWidget" name="MenuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: black;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>931</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Parchment</family>
     <pointsize>126</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Telegraphist</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="helpnav">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>570</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Parchment</family>
     <pointsize>64</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 0px;
    border: 2px solid black;
}
QPushButton:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="translatenav">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>480</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OldNewspaperTypes</family>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 0px;
    border: 2px solid black;
}
QPushButton:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
   </property>
   <property name="text">
    <string>Translate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsnav">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>570</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 0px;
    border: 2px solid black;
}
QPushButton:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="learningnav">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OldNewspaperTypes</family>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 0px;
    border: 2px solid black;
}
QPushButton:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
   </property>
   <property name="text">
    <string>Learn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="practicenav">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>390</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OldNewspaperTypes</family>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 0px;
    border: 2px solid black;
}
QPushButton:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
   </property>
   <property name="text">
    <string>Practice</string>
   </property>
  </widget>
  <widget class="QLabel" name="settingsHelpLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>590</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OldNewspaperTypes</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Change your settings→</string>
   </property>
  </widget>
  <widget class="QLabel" name="learnHelpLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>330</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OldNewspaperTypes</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="text">
    <string>Learn morse code through lessons→</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="practiceHelpLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OldNewspaperTypes</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Practice your skills through typing words→</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="translateHelpLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>490</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OldNewspaperTypes</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Translate morse code to English, or vice versa→</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="referencenav">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>570</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Parchment</family>
     <pointsize>64</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 0px;
    border: 2px solid black;
}
QPushButton:hover {
    background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(255, 255, 255, 0));
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="encodingsHelpLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>590</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>OldNewspaperTypes</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>←View morse encodings for all characters</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
